src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -398,6 +431,68 @@ export function transform(formConfig, form) {
     delete clonedData.providerFacility;
   }
 
+  const incidentKeys = getFlatIncidentKeys();
+
+  const incidents = incidentKeys
+    .filter(incidentKey => clonedData[incidentKey])
+    .map(incidentKey => ({
+      ...clonedData[incidentKey],
+      personalAssault: incidentKey.includes('secondary'),
+      incidentLocation: concatIncidentLocationString(
+        clonedData[incidentKey].incidentLocation,
+      ),
+    }));
+
+  incidentKeys.forEach(incidentKey => {
+    delete clonedData[incidentKey];
+  });
+
+  if (incidents.length > 0) {
+    clonedData.form0781 = {
+      incidents,
+      remarks: clonedData.additionalRemarks781,
+      additionalIncidentText: clonedData.additionalIncidentText,
+      additionalSecondaryIncidentText:
+        clonedData.additionalSecondaryIncidentText,
+      otherInformation: [
```

Comment:
I'm not sure I'm clear on what's happening here; are we passing an array of `ui:title` values for the following properties to `vets-api`? If that's the case, I think we could add a piece of hidden formdata that does essentially the same thing, rather than depend on the uiSchema. @cvalarida any thoughts?

Reply:
Yes, incident `otherInformation` are used as lines in the 781a otherInformation textbox.  We're sending the UI display values instead of the enum values because we want what is actually displayed to the user to make it onto the PDF.

Reply:
With a recent update, we're no longer pulling directly from the `ui:title`. The strings we add in here now just happen to be the same as the titles, which is a great improvement. :tada: